fastf1/api.py

```diff
@@ -1291,6 +1292,78 @@ def session_status_data(path, response=None, livedata=None):
     return data
 
 
+@Cache.api_request_wrapper
+def race_control_messages(path, response=None, livedata=None):
+    """Fetch and parse race control messages.
+
+    Race control messages are sent by race control to all teams to notify of
+    decisions and statuses of the session
+
+    Every message has the following attributes:
+
+        - Utc: Message timestamp
+        - Category (str): Type of message, "Other", "Flag", "Drs", "CarEvent"
+        - Message (str): Content of message
+
+    Other possible attributes are:
+
+        - Status (str): Status of context, e.g. "DISABLED" for disabling DRS
+        - Flag (str): Type of flag being waved "GREEN", "RED", "YELLOW",
+          "CLEAR", "CHEQUERED"
+        - Scope (str): Scope of message "Track", "Sector", "Driver"
+        - Sector (int): Affected track sector for sector-scoped messages
+        - RacingNumber (str): Affected driver for CarEvent messages
+
+    Args:
+        path (str): api path base string (usually ``Session.api_path``)
+        response: Response as returned by :func:`fetch_page` can be passed if
+        it was downloaded already.
+        livedata: An instance of
+        :class:`fastf1.livetiming.data.LiveTimingData` to use as a source
+        instead of the api
+
+    Returns:
+        A dictionary containing one key for each data channel and a list of
+        values per key.
+
+    Raises:
+        SessionNotAvailableError: in case the F1 livetiming api returns no
+        data
```

Comment:
same here

Reply:
Moved back to single line - fits within 79 chars